examples/lfric/eg17/full_example_extract/README.md

```diff
@@ -6,7 +6,8 @@ directory ``../full_example``.
 
 It applies the PSyData extraction transformations to the
 user-supplied kernels (due to #637 we cannot instrument builtins
-at the moment).
+at the moment). See the [PSyKE](https://psyclone.readthedocs.io/en/stable/psyke.html)
+chapter of the PSyclone documentation for details about this transformation.
 
 ## Compilation
 A simple makefile is provided to compile the example. It needs:
```

Comment:
Please insert a blank line between the header and the text.

Reply:
Done.